src/magic_folder/test/test_web.py

```diff
@@ -407,17 +414,25 @@ def test_list_folders(self, folders):
             path_b = path_u.asBytesMode("utf-8")
             path_b.makedirs(ignoreExistingDirectory=True)
 
+        basedir = FilePath(self.mktemp())
         treq = treq_for_folders(
             object(),
-            FilePath(self.mktemp()),
-            AUTH_TOKEN, {
+            basedir,
+            AUTH_TOKEN,
+            {
                 name: magic_folder_config(self.author, FilePath(self.mktemp()), path_u)
                 for (name, path_u)
                 in folders.items()
             },
             False,
         )
 
+        config = load_global_configuration(basedir)
```

Comment:
It took me a minute to realize there is a global configuration at that path as a side-effect of `treq_for_folders`.  Maybe worth a comment here to that effect.